packages/node_modules/@webex/plugin-meetings/src/meeting/util.js

```diff
@@ -673,45 +696,73 @@ MeetingUtil.setPeerConnectionEvents = (meeting) => {
         break;
       case ICE_STATE.COMPLETED:
         LoggerProxy.logger.log('Meeting:util#setPeerConnectionEvents --> ICE STATE COMPLETED.');
-        Metrics.postEvent({event: eventType.ICE_END, meeting});
         break;
       case ICE_STATE.CONNECTED:
+        Metrics.postEvent({event: eventType.ICE_END, meeting});
```

Comment:
Could you please add a comment why `ICE_END` actually means connection established? No one else is going to understand this 😢